ci-operator/step-registry/dpu-operator/workflow/tests/dpu-operator-workflow-tests-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+token_dpu_operator_key=$(cat "/var/run/token/dpu-token/dpu-key")
+endpoint=$(cat "/var/run/token/dpu-token/url")
+
+job_url="https://${endpoint}/job/Lab144_DPU_Operator_Test/lastBuild"
+
+while : 
+do
+    job_check=$(curl -k -s --resolve "${endpoint}:443:10.0.180.88" "$job_url/api/json")
```

Comment:
What is `10.0.180.88`?

Reply:
it is the address we want our endpoint to resolve to

Reply:
Does this mean that our internal jenkins URL has no DNS entry from the PROW CI point of view? I.e. the name cannot be resolved to an IP address?

Reply:
yes the name of the endpoint cannot be resolved to an ip address on it's own.

Reply:
Please also document this by commenting. You can store ""${endpoint}:443:10.0.180.88"" in a variable. However i am puzzled why the name can't be resolved, can any name be resolved from our network?

Reply:
I am not sure... I have not tested this